templates/observability-remote-write-proxy-route.yml

```diff
@@ -0,0 +1,23 @@
+# This file contains an OpenShift template that creates a route helping to
+# directly access the Observability Remote Write Proxy in the
+# development environment.
```

Comment:
This route is for development only. For our stage and production environment different routes will be created, from different OCP templates located in another repository